docs/api.mdx

```diff
@@ -184,7 +184,7 @@ This endpoint allows users to query for a list of all addresses that hold any Gi
 
 ### `GET /v1/address/:address/gitpoaps`
 
-This endpoint allows users to query for some address's (either and ETH or ENS address) GitPOAPs. This returns data like:
```

Comment:
I think this should stay as is since without the parenthetical, it would read "address's GitPOAPs"

Reply:
Ohh, just changed that one!